main.js

```diff
@@ -6,17 +6,17 @@ import { getFeedbacks } from './feedbacks.js'
 import UpgradeScripts from './upgrades.js'
 
 import fetch from 'node-fetch'
-import { XMLParser, XMLBuilder } from 'fast-xml-parser'
+import { /*XMLParser,*/ XMLBuilder } from 'fast-xml-parser'
 import dayjs from 'dayjs'
 import duration from 'dayjs/plugin/duration.js'
 dayjs.extend(duration)
 
-const parser = new XMLParser({
+/* const parser = new XMLParser({
```

Comment:
I could not find this being used anyway - is it legacy?

Reply:
It is legacy and can be removed.